### PR TITLE
Makes the SM surge event even more powerful

### PR DESCRIPTION
**Pull request by patpol4**
# General Documentation

### Intent of your Pull Request

After the latest changes it's still wet farts. What I did is raise the maximum random powerup to as high as 100 000.
This may seem like a lot compared to what we have now(8k) but from what I understand, the way this code works is just set it to the power value for a split second, making even values up to 10k insignificant on a default engine setup.
Made it a bit more rare to balance it out.

Tested it by getting a simple straight pipe setup and setting the SM power level to 100k via varediting. What happened was it just burned for a little while but didn't even start delaminating.

### Why is this change good for the game?

As it is right now it still does nothing and only fills up the events.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
What I did is raise the maximum random powerup to as high as 100 000.
This may seem like a lot compared to what we have now(8k) but from what I understand, the way this code works is just set it to the power value for a split second, making even values up to 10k insignificant on a default engine setup.
Made it a bit more rare to balance it out.

### What should players be aware of when it comes to the changes your PR is implementing?
The SM surge event is now a bit more rare but more powerful

### What general grouping does this PR fall under? 
Engineering

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
Not really

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
The earliest start for it is 10 minutes in and the SM crystal has to has been powered.

# Changelog

:cl:  
tweak: Makes the SM surge event even more powerful, lowers its chance to appear
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
Would be nice to at least get this testmerged to see if I'm not a giant idiot and got this all wrong.

**Comment by Jarod1200** [comment]
More power = good

**Comment by patpol4** [comment]
Ok it turns out I am, the values should still be fine though.

File: code/modules/events/supermatter_surge.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/supermatter_surge
 	name = "Supermatter Surge"
 	typepath = /datum/round_event/supermatter_surge
-	weight = 20
+	weight = 15
```
Comment (alexkar598) [inline code/modules/events/supermatter_surge.dm:4]: Undocumented change.

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/events/supermatter_surge.dm [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	var/power = 2000
 
 /datum/round_event/supermatter_surge/setup()
-	power = rand(400,8000)
+	power = rand(1000,100000)
```
Comment (alexkar598) [inline code/modules/events/supermatter_surge.dm:16]: Thats a very wide range, this will lead to inconsistent gameplay.

File: code/modules/events/supermatter_surge.dm [hunk 3]
```diff
@@ -13,7 +13,7 @@
...
 
 /datum/round_event/supermatter_surge/announce()
 	if(power > 800 || prob(round(power/8)))
```
Comment (alexkar598) [inline code/modules/events/supermatter_surge.dm]: 1) Both these conditions are redundant as the power will never be under 800 and the prob will never fail as the minimum is 1000/8 so prob(125).
2) The class number can be ridiculously high now. Anywhere from class 3 to class 201

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/events/supermatter_surge.dm (hunk 1 above)
Comment (patpol4) [reply code/modules/events/supermatter_surge.dm:4]: its in the description but i did the changelog too now

File: code/modules/events/supermatter_surge.dm (hunk 2 above)
Comment (patpol4) [reply code/modules/events/supermatter_surge.dm:16]: Yes and the severity is going to be stated in the announcement so it's not like it's secret and people don't realize how bad it is.
Do you think i should raise the minimum?

File: code/modules/events/supermatter_surge.dm (hunk 3 above)
Comment (patpol4) [reply code/modules/events/supermatter_surge.dm]: 1. removed the condition so it always announces,
2. yes

**Comment by MegaEmpirical** [comment]
Yessssssssss
